lacci/lib/shoes/app.rb

```diff
@@ -99,33 +100,38 @@ def initialize(
 
       # Try to de-dup as much as possible and not send repeat or multiple
       # destroy events
-      @watch_for_destroy = bind_shoes_event(event_name: "destroy") do
+      @watch_for_destroy = bind_shoes_event(event_name: 'destroy') do
         Shoes::DisplayService.unsub_from_events(@watch_for_destroy) if @watch_for_destroy
         @watch_for_destroy = nil
-        self.destroy(send_event: false)
+        destroy(send_event: false)
       end
 
-      @watch_for_event_loop = bind_shoes_event(event_name: "custom_event_loop") do |loop_type|
-        raise(Shoes::Errors::InvalidAttributeValueError, "Unknown event loop type: #{loop_type.inspect}!") unless CUSTOM_EVENT_LOOP_TYPES.include?(loop_type)
+      @watch_for_event_loop = bind_shoes_event(event_name: 'custom_event_loop') do |loop_type|
+        unless CUSTOM_EVENT_LOOP_TYPES.include?(loop_type)
+          raise(Shoes::Errors::InvalidAttributeValueError,
+                "Unknown event loop type: #{loop_type.inspect}!")
+        end
 
         @event_loop_type = loop_type
       end
 
-      Signal.trap("INT") do
-        @log.warn("App interrupted by signal, stopping...")
+      Signal.trap('INT') do
+        @log.warn('App interrupted by signal, stopping...')
         puts "\nStopping Shoes app..."
         destroy
       end
     end
 
     def init
-      send_shoes_event(event_name: "init")
+      send_shoes_event(event_name: 'init')
       return if @do_shutdown
 
       with_slot(@document_root) do
         @content_container = flow(width: 1.0, height: 1.0)
         with_slot(@content_container, &@app_code_body)
       end
+
+      render_index_if_defined_on_first_boot
```

Comment:
I tried a few homes for this, which seemed to break and not make sense. Hopefully this is the best home

Reply:
This looks reasonable to me on first blush.